cmd/vulcan-http-headers/Dockerfile

```diff
@@ -15,7 +15,7 @@ WORKDIR /opt/
 # Install the HTTP Observatory
 RUN git clone https://github.com/mozilla/http-observatory
 WORKDIR /opt/http-observatory
-RUN git reset --hard e3ef44d9ef3a0d6147b55934e81bd67fd7e2bd7e
+RUN git reset --hard 0b7928ee9d09a3f5dffdb81f27d3e5f80ef2024c
```

Comment:
Please upgrade to 53f07f3e70801fb6a05aa1b38662d75abc4efa9e

Reply:
fixed